### PR TITLE
GA Claude 3.5 Sonnet and Retrieve Provider Payload

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

File: fern/openapi/openapi.yml [hunk 1]
```diff
@@ -1798,51 +1798,6 @@ components:
           nullable: true
       required:
       - model_name
-    BulkCreateTestSuiteTestCaseDataRequest:
```
Comment (dvargas92495) [inline fern/openapi/openapi.yml:1801]: cc: @noanflaherty to confirm these changes

File: fern/openapi/openapi.yml [hunk 2]
```diff
@@ -2964,18 +2950,19 @@ components:
...
       type: object
+      description: A value representing an array of Vellum variable values.
       properties:
         id:
           type: string
           description: The variable's uniquely identifying internal id.
         name:
           type: string
         type:
-          type: string
```
Comment (dvargas92495) [inline fern/openapi/openapi.yml:2974]: Expected from previous work

File: fern/openapi/openapi.yml [hunk 3]
```diff
@@ -3976,6 +3970,30 @@ components:
         compute_nanos:
           type: integer
           nullable: true
+    MapEnum:
```
Comment (dvargas92495) [inline fern/openapi/openapi.yml:3973]: cc: @m-abboud to confirm that these schemas look ok

File: fern/fern.config.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
   "organization": "vellum",
-  "version": "0.30.0"
-}
+  "version": "0.31.0-rc0"
```
Comment (siddseethepalli) [inline fern/fern.config.json:3]: Nice

File: fern/fern.config.json [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+}
```
Comment (siddseethepalli) [inline fern/fern.config.json:4]:
```suggestion
}

```

**Review by siddseethepalli — APPROVED** [review]
Claude 3.5 changes lgtm!

This is a good example where splitting into multiple PRs would expedite review/merge of the orthogonal components

**Pull request merged**